pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java

```diff
@@ -124,7 +124,12 @@ private static Provider loadConscryptProvider() {
             conscryptClazz = Class.forName("org.conscrypt.Conscrypt");
             conscryptClazz.getMethod("checkAvailability").invoke(null);
         } catch (Throwable e) {
-            log.warn("Conscrypt isn't available. Using JDK default security provider.", e);
+            if (e.getCause() != null && e.getCause().getClass().getName().equals("java.lang.UnsatisfiedLinkError")) {
```

Comment:
Could we use `debug` instead of `warn`?

Reply:
It's better to use level `warn` because it is a exception — not a `If - else` or something needs to record. This `warn` means conscrypt is not supported which needs to remind.

Reply:
If you want to avoid the too-long message, I suggest you just output the message and then add a debug log to print the stack trace.